modules/special-rules/pages/pulsar-x.adoc

```diff
@@ -0,0 +1,19 @@
+= Pulsar (X)
+
+This weapon is unique to the Eldar race, and it demonstrates their mastery of laser weaponry.
+
+The weapon shoots in rapid but powerful bursts, guided by advanced targeting technologies.
+As a result, the weapon can destroy multiple targets in a single salvo.
+
+---
+
+When you shoot with a {pulsar-x}:
+
+. Determine the number of shots, based on the value of `(X)` that we specify — for example, _`(D3+1)`_ or _`(D6)`_.
+. Resolve that many {macro-weapon} shots against the target detachment.
```

Comment:
To tie in with my other suggestions, I think we need to reframe things here around the idea of a "salvo of {macro-weapon} shots". Perhaps something like:

```suggestion
IMPORTANT: A {pulsar-x} shot results in a salvo of {macro-weapon} shots -- but it still only counts as a single super-heavy weapon.

. Determine the number of shots in the salvo, based on the value of `(X)` that we specify — for example, _`(D3+1)`_ or _`(D6)`_.
. Resolve that many {macro-weapon} shots against the target detachment.
```
...and then we can delete the note that we currently have below the bullets.